src/Splunk.OpenTelemetry.AutoInstrumentation/Traces.cs

```diff
@@ -17,11 +17,13 @@
 using System;
 using OpenTelemetry.Exporter;
 using OpenTelemetry.Trace;
+using Splunk.OpenTelemetry.AutoInstrumentation.Logging;
 
 namespace Splunk.OpenTelemetry.AutoInstrumentation;
 
 internal class Traces
 {
+    private static readonly ILogger Log = OtelLogging.GetLogger();
```

Comment:
Have you considered calling `OpenTelemetry.AutoInstrumentation.Logging.OtelLogging` using reflection?